Cargo.toml

```diff
@@ -29,7 +29,7 @@ swc_ecma_codegen = { git = "https://github.com/ef4/swc.git", branch = "content-t
 swc_ecma_ast = { git = "https://github.com/ef4/swc.git", branch = "content-tag" }
 swc_ecma_visit = { git = "https://github.com/ef4/swc.git", branch = "content-tag" }
 swc_ecma_utils = { git = "https://github.com/ef4/swc.git", branch = "content-tag" }
-swc_ecma_transforms = { git = "https://github.com/ef4/swc.git", branch = "content-tag" }
+swc_ecma_transforms_base = { git = "https://github.com/ef4/swc.git", branch = "content-tag" }
```

Comment:
This change isn't strictly necessary here, but the `hygiene`, `resolver` etc are all actually defined in `swc_ecma_transforms_base` and just re-exported from `swc_ecma_transforms`. There are some extra stuff (`renamer`, etc) that I may need to actually fix #71 so keeping it here.